<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_AddStudent</class>
 <widget class="QDialog" name="Dialog_AddStudent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>221</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>220</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Fira Code</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Add Student</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources/res.qrc">
    <normaloff>:/img/logo.ico</normaloff>:/img/logo.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="2" column="0">
      <widget class="QLabel" name="sid">
       <property name="font">
        <font>
         <family>Fira Code</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>SID</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="4">
      <widget class="QLineEdit" name="le_add_sid"/>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="class_2">
       <property name="font">
        <font>
         <family>Fira Code</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Class</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="name">
       <property name="font">
        <font>
         <family>Fira Code</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QSpinBox" name="sbox_add_age"/>
     </item>
     <item row="1" column="4">
      <widget class="QLineEdit" name="le_add_class"/>
     </item>
     <item row="3" column="1" colspan="4">
      <widget class="QLineEdit" name="le_add_phone"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="le_add_grade"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="grade">
       <property name="font">
        <font>
         <family>Fira Code</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Grade</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="3" colspan="2">
      <widget class="QPushButton" name="btn_cancelAdd">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="age">
       <property name="font">
        <font>
         <family>Fira Code</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Age</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="1" colspan="4">
      <widget class="QLineEdit" name="le_add_wechat"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="wechat">
       <property name="font">
        <font>
         <family>Fira Code</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Wechat</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="le_add_name"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="phone">
       <property name="font">
        <font>
         <family>Fira Code</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Phone</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QPushButton" name="btn_saveAdd">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>le_add_name</tabstop>
  <tabstop>sbox_add_age</tabstop>
  <tabstop>le_add_grade</tabstop>
  <tabstop>le_add_class</tabstop>
  <tabstop>le_add_sid</tabstop>
  <tabstop>le_add_phone</tabstop>
  <tabstop>le_add_wechat</tabstop>
  <tabstop>btn_saveAdd</tabstop>
  <tabstop>btn_cancelAdd</tabstop>
 </tabstops>
 <resources>
  <include location="Resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
